src/scripts/glue-etl.py

```diff
@@ -70,22 +73,35 @@ def dump_df_to_s3(df, objectName, header = True):
         format_options={"writeHeader": header},
         format="csv")
 
+def dump_df_to_s3_for_bulk_load(df, objectName, header = True):
+    objectKey = f"{args['bulk_load_prefix']}/{objectName}.csv"
```

Comment:
it’s not a right place to load prefix. dumping the files to `processed dir`, then passing the path to `load properties` task in pipeline

you can refer to how training job to consume the exported csv files